@3rdweb-sdk/react/hooks/useApi.ts

```diff
@@ -302,6 +305,19 @@ export function useConfirmEmail() {
         throw new Error(json.message);
       }
 
+      // Add account with validated email to the Paper database as a seller
```

Comment:
I wonder if we could do this from server -> server?

is there something that requires this to happen from the client?

Reply:
Moved to api-server 👍